sdk/communication/communication-rooms/package.json

```diff
@@ -0,0 +1,133 @@
+    "@azure/core-auth": "^1.3.2",
+    "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/logger": "^1.0.0",
+    "@types/uuid": "^8.3.4",
+    "tslib": "^1.9.3",
+    "uuid": "^8.3.2",
+    "@azure/core-client": "^1.6.0",
+    "@azure/core-rest-pipeline": "^1.3.0"
+  },
+  "keywords": [
+    "node",
+    "azure",
+    "typescript",
+    "browser",
+    "isomorphic",
+    "Azure",
+    "communication",
+    "rooms",
+    "cloud"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "types": "types/communication-rooms.d.ts",
+  "devDependencies": {
+    "@azure/communication-identity": "^1.1.0-beta.2",
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/logger": "~1.0.4",
+    "@microsoft/api-extractor": "^7.18.11",
+    "@rollup/plugin-commonjs": "11.0.2",
+    "@rollup/plugin-json": "^4.0.0",
+    "@rollup/plugin-multi-entry": "^3.0.0",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "cross-env": "^7.0.2",
+    "dotenv": "^8.2.0",
+    "eslint": "^8.0.0",
+    "mkdirp": "^1.0.4",
+    "rollup": "^1.16.3",
+    "rollup-plugin-sourcemaps": "^0.4.2",
+    "tslib": "^2.4.0",
+    "typescript": "~4.2.0",
+    "uglify-js": "^3.4.9",
+    "rimraf": "^3.0.0",
+    "@azure-tools/test-credential": "~1.0.0",
+    "@azure/test-utils": "~1.0.0",
+    "mocha": "~10.0.0",
+    "@types/chai": "~4.3.1",
+    "@types/sinon": "~10.0.12",
+    "karma": "^6.3.20",
+    "karma-env-preprocessor": "^0.1.1",
+    "chai": "^4.2.0",
+    "@types/mocha": "^7.0.2",
+    "sinon": "~14.0.0",
+    "@azure-tools/test-recorder": "^2.0.0",
+    "@azure/core-util": "~1.0.1"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/communication/communication-rooms/",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+ "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/communication-rooms.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "scripts": {
+    "build": "npm run clean && tsc -p . && dev-tool run bundle && api-extractor run --local",
+    "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/index.js.map'\" -o ./dist/index.min.js ./dist/index.js",
+    "prepack": "npm run build",
+    "pack": "npm pack 2>&1",
+    "extract-api": "api-extractor run --local",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "audit": "echo skipped",
+    "clean": "rimraf dist dist-browser dist-esm test-dist temp types *.tgz *.log",
+    "build:node": "tsc -p . && dev-tool run bundle",
+    "build:browser": "tsc -p . && dev-tool run bundle",
+    "build:test": "tsc -p . && dev-tool run bundle",
+    "build:samples": "dev-tool samples publish --force",
+    "check-format": "echo skipped",
+    "execute:samples": "dev-tool samples run samples-dev",
+    "format": "echo skipped",
```

Comment:
This shouldn't be skipped. Please take a look at sdk/template/template/package.json and make sure that the script mirror it as closely as possible. Remove any scripts not present there